magento/adobe-stock-integration
# Fix add Selected button, to abjust working with wysiwyg and widgets

### Description (*)


### Fixed Issues (if relevant)

1. magento/adobe-stock-integration#1068: The Image from Enhanced Media Gallery is not added via WYSIWYG

### Manual testing scenarios (*)

1. Installed Magento with Adobe Stock Integration
2. Configured integration in Stores -> Configuration -> Advanced-> System -> Adobe Stock Integration fieldset
3. Enable Enhanced Media Gallery in Stores -> Configuration -> Advanced -> System
4. From Admin go to Content - Pages, click Add New Page
5. Expand Content
6. Click Insert/edit Image
7.Click Search source
**The Image
![DeepinScreenshot_select-area_20200331131050](https://user-images.githubusercontent.com/2481206/78014750-0e703000-7351-11ea-856b-e797756ae5ac.png)
 is added and a link appears in the Source field.**

**Scenario 2**
1. From Admin go to Content - Pages, click Add New Page
2. Expand Content
3. Click Show/Hide editor 
4.Click Insert Image 
**Proper img tag generated, and image is displayed after switch to the wysiwyg**

![DeepinScreenshot_select-area_20200331130732](https://user-images.githubusercontent.com/2481206/78014466-a4f02180-7350-11ea-8f87-60abae03c71c.png)
![DeepinScreenshot_select-area_20200331130821](https://user-images.githubusercontent.com/2481206/78014505-b1747a00-7350-11ea-9b79-9aed795474b7.png)
